opendocdb-cts/.golangci.yml

```diff
@@ -117,7 +117,7 @@ linters-settings:
       - name: add-constant
         disabled: true # use common sense during review
       - name: cognitive-complexity
-        arguments: [25]
+        arguments: [30]
```

Comment:
Could we revert this change https://github.com/OpenDocDB/cts/pull/23#discussion_r2033115516?

Reply:
Now, it's not :)